### PR TITLE
Handful of matching funcs, starrod_to_c.py in /tools for converting starrod enums to c headers

### PR DESCRIPTION
**Pull request by JoshW-7**

### CODE REVIEW

File: papermario/include/items.h [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+	HAMMER1_ICON = 0x0166,
+	HAMMER2_ICON = 0x0167,
+	HAMMER3_ICON = 0x0168,
+	BOOTS1_ICON = 0x0169,
+	BOOTS2_ICON = 0x016A,
+	BOOTS3_ICON = 0x016B,
+	ITEMS_ICON = 0x016C
+} eItem;
+
+#endif
```
Comment (ethteck) [inline papermario/include/items.h]: Newline at end of files please

File: papermario/src/code_1b40_len_20b0.c [hunk 2]
```diff
@@ -32,7 +32,7 @@ extern int D_802EB3D0[];
 extern u8 D_80074021;
 extern s32 D_8009A5D8;
 
-extern s32 D_8009A650[1];
+//extern s32 D_8009A650[1]; Declared in variables.h
```
Comment (ethteck) [inline papermario/src/code_1b40_len_20b0.c]: I think this comment isn't necessary - people can just search the project for this symbol if they wanna see where it's externed

File: papermario/src/code_fa4c0_len_3bf0.c [hunk 3]
```diff
@@ -104,16 +103,80 @@ s32 RemoveItemAt(script_context* script, s32 initialCall) {
...
+        return 2;
+    }
+
+    for(i=0; i < ARRAY_COUNT(playerData->keyItems); i++) {
+        if (playerData->keyItems[i] == 0) {
+            break;
+        }
+    }
+
+    if (i < 32) {
```
Comment (pixel-stuck) [inline papermario/src/code_fa4c0_len_3bf0.c]: ``if (i < ARRAY_COUNT(playerData->keyItems) {``

Comment (ethteck) [inline papermario/src/code_fa4c0_len_3bf0.c]: ARRAY_COUNT macro here too please

File: papermario/src/code_fa4c0_len_3bf0.c [hunk 4]
```diff
@@ -104,16 +103,80 @@ s32 RemoveItemAt(script_context* script, s32 initialCall) {
...
+    }
+    return 2;
+}
 
-INCLUDE_ASM(code_fa4c0_len_3bf0, HasKeyItem);
+s32 func_802D6954(void) {
+    func_800E01A4();
+    disable_player_physics();
+    func_800EF600();
+    D_8009A650[0] &= -0x41;
```
Comment (pixel-stuck) [inline papermario/src/code_fa4c0_len_3bf0.c]: bitwise AND-ing with a negative is kinda weird. Use ~0x40 instead.

**Comment by pixel-stuck** [comment]
With latest changes, lgtm

**Pull request merged**